package.json

```diff
@@ -29,15 +29,15 @@
     "vinyl-fs": "2.3.1"
   },
   "devDependencies": {
-    "@elastic/eslint-config-kibana": "^0.13.0",
-    "babel-eslint": "^7.2.3",
-    "eslint": "^4.1.0",
+    "@elastic/eslint-config-kibana": "^0.15.0",
+    "babel-eslint": "^8.1.2",
+    "eslint": "^4.14.0",
     "eslint-plugin-babel": "^4.1.1",
     "eslint-plugin-import": "^2.6.0",
-    "eslint-plugin-jest": "^21.0.0",
+    "eslint-plugin-jest": "^21.5.0",
```

Comment:
The eslint dependencies are "outdated" on purpose, to match upstream. See https://github.com/elastic/kibana/blob/master/packages/eslint-config-kibana/package.json

Updating the config and babel-eslint should be fine though.

Reply:
Right, but we don't use yarn in Kibana so `^21.0.0` resolves to `21.5.0`, but since we use yarn here it doesn't.

Reply:
Also, we pin to 21.5.0 in Kibana https://github.com/elastic/kibana/blob/master/package.json#L237 but only require `^21.0.0` as a peer dep in the package

Reply:
I removed the yarn lock file from the repo, it was doing more harm than good.

Reply:
Also, yarn doesn't recursively use lock files to control dependency version (or didn't, did that change?), so this shouldn't affect users when they go to install the module anyway, right?

Reply:
Well, they're dev deps so the yarn.lock file would still apply, but if there isn't a yarn.lock file then none of this is necessary :)

Reply:
I guess I don't care about the eslint package version here much anyway, all the peerDeps upstream use the caret anyway...